client/chainlink_models.go

```diff
@@ -1,25 +1,33 @@
 package client
 
-import (
-	"bytes"
-	"fmt"
-	"text/template"
-	"time"
+type TransactionsData struct {
+	Data []TransactionData    `json:"data"`
+	Meta TransactionsMetaData `json:"meta"`
+}
 
-	"gopkg.in/guregu/null.v4"
-)
+type TransactionData struct {
+	Type       string                `json:"type"`
+	ID         string                `json:"id"`
+	Attributes TransactionAttributes `json:"attributes"`
+}
 
-// EIServiceConfig represents External Initiator service config
-type EIServiceConfig struct {
-	URL string
+type TransactionAttributes struct {
+	State    string `json:"state"`
```

Comment:
TransactionAttributes/TransactionsData/TransactionData/TransactionsMetaData can be moved too because they are ETH specific.

Reply:
Discussed offline. These are all used in testreports.